cilium-cli/connectivity/check/test.go

```diff
@@ -949,16 +833,6 @@ func (t *Test) collectSysdump() {
 func (t *Test) ForEachIPFamily(do func(features.IPFamily)) {
 	ipFams := []features.IPFamily{features.IPFamilyV4, features.IPFamilyV6}
 
-	// The per-endpoint routes feature is broken with IPv6 on < v1.14 when there
-	// are any netpols installed (https://github.com/cilium/cilium/issues/23852
-	// and https://github.com/cilium/cilium/issues/23910).
-	if f, ok := t.Context().Feature(features.EndpointRoutes); ok &&
-		f.Enabled && (len(t.cnps) > 0 || len(t.knps) > 0) &&
-		versioncheck.MustCompile("<1.14.0")(t.Context().CiliumVersion) {
-
-		ipFams = []features.IPFamily{features.IPFamilyV4}
-	}
-
```

Comment:
This part (and the other `skip if <v1.14` part below) should have *really* been a separate patch. And ideally even a separate PR, so that the impact on downgrade testing for v1.14 -> v1.13 would have been obvious.